src/integer.c

```diff
@@ -0,0 +1,120 @@
+#include <bcc/tokens.h>
+#include <bcc/scanner.h>
+
+char * itoa(size_t num)
```

Comment:
Maybe use a library for this?

Reply:
Again, that would be my preference (but for some reason I couldn't link the library in -- we can patch it later, I guess).